mlrun/datastore/utils.py

```diff
@@ -180,3 +181,55 @@ def get_kafka_brokers_from_dict(options: dict, pop=False) -> typing.Optional[str
             FutureWarning,
         )
     return kafka_bootstrap_servers
+
+
+def revert_list_filters_to_tuple(additional_filters):
```

Comment:
Revert is a bit confusing here. Maybe
```suggestion
def transform_list_filters_to_tuple(additional_filters):
```